lib/ui/form/form_screen.dart

```diff
@@ -74,28 +87,36 @@ class FormScreenState extends ConsumerState<FormScreen> {
           if (surveyDetails != null) ...[
             DimmedBackground(background: surveyDetails.coverImageUrl),
             PageView.builder(
-              // TODO: Integrate item count from survey details #25
-              itemCount: 3,
-
+              itemCount: 1 + questionTotal,
               controller: _pageController,
+              physics: const NeverScrollableScrollPhysics(),
               itemBuilder: (BuildContext context, int index) {
                 if (index == 0) {
                   return FormSurveyDetailPage(
                     surveyDetails: surveyDetails,
                   );
                 } else {
-                  return const FormSurveyQuestionPage();
+                  return FormSurveyQuestionPage(
+                    questionIndex: index,
+                    questionTotal: questionTotal,
+                  );
                 }
               },
+              onPageChanged: (int index) {
+                setState(() {
+                  _showStartSurveyButton = index == 0;
+                  _showNextSurveyButton = index > 0 && index < questionTotal;
+                  _showSubmitSurveyButton = index == questionTotal;
+                  _showCloseButton = index != 0;
+                });
+              },
             ),
-            _buildStartButton()
+            _buildCloseButton(context),
+            _buildStartSurveyButton(),
+            _buildNextSurveyButton(),
+            _buildSubmitSurveyButton()
```

Comment:
Should we wrap `Align(alignment: Alignment.bottomRight,` here instead of on each button?